operator/bundle_helpers/patch-csv.py

```diff
@@ -70,6 +70,8 @@ def patch_csv(csv_doc, version, operator_image, first_version, no_related_images
     csv_doc["metadata"]["annotations"]["olm.skipRange"] = f'>= {x}.{y-1}.0 < {version}'
 
     # multi-arch
+    if "labels" not in csv_doc["metadata"]:
+        csv_doc["metadata"]["labels"] = {}
```

Comment:
Why is this necessary?

Reply:
Because `labels` key does not exist in `metadata` map since I removed default `amd64` there.